presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -125,20 +126,23 @@ public BaseJdbcClient(BaseJdbcConfig config, String identifierQuote, ConnectionF
                 identifierQuote,
                 connectionFactory,
                 requireNonNull(config, "config is null").isCaseInsensitiveNameMatching(),
-                config.getCaseInsensitiveNameMatchingCacheTtl());
+                config.getCaseInsensitiveNameMatchingCacheTtl(),
+                config.getJdbcTypesMappedToVarchar());
```

Comment:
move the new arg right after `identifierQuote` and `connectionFactory` (same in assignments below)
(i'm aware the `identifierQuote` and `connectionFactory` are misordered)